src/Coordinates/Location.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "Vector2.hpp"
+
+struct Location : Vector2<float> {
+	Location(float x, float y) : Vector2<float>(x, y){};
+};
+
```

Comment:
```suggestion
using Location = Vector2<float>;
```

Reply:
This ended up going away as I fleshed out Vector2 and Size (commits before this had operator overloads in Location, before I created Vector2 base class, and pulled the functions into there).

This commit must have been while I was still working on that.